ts/parts/SvgRenderer.ts

```diff
@@ -6087,6 +6087,9 @@ extend(SVGRenderer.prototype, /** @lends Highcharts.SVGRenderer.prototype */ {
                     anchorY: anchorY
                 });
             }
+
+            updateBoxSize();
+            updateTextPadding();
```

Comment:
The `updateBoxSize` and `updateTextPadding` functions are called from `wrapper.textSetter`, which should be invoked from line 6079. Why do we need to call them again here?

Reply:
Oh, right, I missed that! I thought updating box/padding was not called on add.